Gemfile.lock

```diff
@@ -84,7 +84,7 @@ GEM
       activesupport (= 4.1.1)
       rake (>= 0.8.7)
       thor (>= 0.18.1, < 2.0)
-    rake (10.3.2)
+    rake (10.5.0)
```

Comment:
Hey, we're actually locked on version 10.3.2 of rake for `reasons`. Please revert this change!